frontend/utilities/helpers.ts

```diff
@@ -59,37 +59,6 @@ const labelSlug = (label: ILabel): string => {
   return `labels/${id}`;
 };
 
-const labelStubs = [
```

Comment:
are you sure this is ok to remove? I think this is needed for the label filter dropdown. I could be wrong but remember running into this array when building that feature

Reply:
It isn't referenced anywhere and was making it so labels/new was prefilled because there was a label stub for new. I double checked the label filter dropdown for functionality.

Reply:
on main branch its used in [formatLabelResponse](https://github.com/fleetdm/fleet/blob/main/frontend/utilities/helpers.ts#L253)  which is then used in [entities/labels](https://github.com/fleetdm/fleet/blob/8a242f7ba19fb6ed97868a1729c998515cf7103d/frontend/services/entities/labels.ts#L48). tbh not sure it is needed but yah lets just check functionality around filtering by status and label still works